pkg/rpc/nodedialer/nodedialer_test.go

```diff
@@ -186,7 +186,6 @@ func TestConnHealthTryDial(t *testing.T) {
 
 	// When no connection exists, we expect ConnHealthTryDial to dial the node,
 	// which will return ErrNoHeartbeat at first but eventually succeed.
-	require.Equal(t, rpc.ErrNotHeartbeated, nd.ConnHealthTryDial(staticNodeID, rpc.DefaultClass))
```

Comment:
Thanks, makes sense. But let's make sure the connection is actually closed first, by using `ConnHealth`:

```
  // Make sure no connection exists yet, via ConnHealth().
	require.Equal(t, rpc.ErrNoConnection, nd.ConnHealth(staticNodeID, rpc.DefaultClass))
```

Reply:
Understood!